client/components/product-card/style.scss

```diff
@@ -204,10 +204,16 @@
 	line-height: 20px;
 	color: var( --color-text-subtle );
 
-	a {
+	a:not( .button ) {
 		@extend %product-card-link-plain;
 	}
-
+	.button {
+		margin-left: auto;
+		margin-right: auto;
+		display: block;
+		max-width: 200px;
+		text-align: center;
```

Comment:
This is not necessary.

Reply:
I tried it without and it didn't work.

Reply:
Can you explain why is it necessary? If we don't specify width of the buttons like I suggested, it will not be necessary.